code/orders-api-rest/api/openapi-rest.yml

```diff
@@ -1056,6 +1056,12 @@ components:
           description: Indicates whether the order has been paid
         orderStatus:
           $ref: '#/components/schemas/OrderStatus'
+        availableStatuses:
+          type: array
+          items:
+            type: string
```

Comment:
I think you can use OrderStatus component

Reply:
Yes, i will reuse this component